libs/designsystem/src/lib/components/modal/footer/modal-footer.component.html

```diff
@@ -1,3 +1,3 @@
-<ion-footer>
+<ion-footer [class.light-bg]="themeColor && themeColor === 'light'">
```

Comment:
Consider using the theme-color directive instead, so we don't need theme-color logic inside individually components like avatar, badge, card etc.